contentcuration/contentcuration/static/js/edit_channel/channel_set/views/ChannelSelectList.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div class="channelList">
+    <div class="channelListHeader uppercase" @click="toggleChannelList">
+      {{translateName}}
+      <span class="toggler material-icons">{{togglerClass}}</span>
+    </div>
+    <div v-show="isExpanded" class="channelListContent">
+      <em v-if="isLoading" class="default-item">
+        {{ $tr('channelLoadingText') }}
+      </em>
+      <div v-else-if="channels.length === 0" class="default-item">
+        {{ $tr('noChannelsText') }}
+      </div>
+      <div class="container-fluid">
+        <div class="channelCountText">
+          {{ $tr("publishedChannelCount", {"channelCount": channels.length}) }}
+        </div>
+        <ChannelItem
+          v-for="channel in channels"
+          :key="channel.id"
+          :channel="channel"
+        />
+      </div>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+import { mapGetters, mapActions } from 'vuex';
+import ChannelItem from './ChannelItem.vue';
+
+export default {
+  name: 'ChannelSelectList',
+  $trs: {
+    EDIT: "My Channels",
```

Comment:
If you are mapping the keys here to a constant, might be better to import the constant definition and then use programmatic key definition:
```
$trs: {
    [Constants.EDIT]: "My Channels",
```
This should give the same net result, but is not contingent on the actual value of the constant itself.